receiver/awscontainerinsightreceiver/README.md

```diff
@@ -55,6 +55,10 @@ The "PodName" attribute is set based on the name of the relevant controllers lik
 
 The "FullPodName" attribute is the pod name including suffix. If false FullPodName label is not added. The default value is false
 
+**cluster_name (optional)**
+
+"ClusterName" can be used to explicitly provide the cluster's name for scenarios where it's not possible to auto-detect it using EC2 tags.
```

Comment:
super minor nitpick - IMO I think the use case is clearer by inverting this, instead of saying "not EKS on EC2". Idk.

Reply:
Do you mean just rephrase the comment? Any suggestions on what would be better?

Reply:
Off the top of my head, not sure. I think my pain point is the "where it's not possible". When is it not possible to detect EC2 tags? Intuitively, that would be when running on-prem k8s, but that also would include something like Fargate. Which is why I was wondering if it makes sense to be more explicit that this is when running the agent in a context that is NOT EKS on EC2.

Reply:
That makes sense. I can update it to be more explicit.